Cargo.toml

```diff
@@ -298,7 +298,11 @@ base64 = '0.22'
 bincode = '1'
 bit_field = '0.10'
 bitvec = '1'
-bls12_381 = { git = 'https://github.com/grandinetech/universal-precompiles.git', tag = 'bls12_381-6bb9695-up.1' }
+# bls12_381 = { git = 'https://github.com/grandinetech/universal-precompiles.git', tag = 'bls12_381-6bb9695-up.1' }
+# bls12_381 = { package = "sp1_bls12_381", path = "../zisk-bls12_381" }
+# bls12_381 = { path = "../grandine-universal-precompiles-bls12_381" }
+bls12_381 = { git = 'https://github.com/jimmychu0807/grandine-universal-precompiles.git', branch = 'jc/zkvm-zisk-bls12_381' }
+
```

Comment:
putting here this comment just as a reminder to change it once precompiles are merged